styleguide/gulpfile.js

```diff
@@ -268,7 +276,7 @@ gulp.task('default', ['clean:before'], function (callback) {
 
   // We need to re-run sass last to make sure the latest styles.css gets loaded
   runSequence(
-    ['scripts', 'fonts', 'images', 'sass'],
```

Comment:
Perhaps we can number reset.scss so it loads in front of all the other scss files or put it in the mixins directory because that loads first?

Reply:
I think having it in the HTML is a good move so that it is completely separate from the added styles.